crawl-ref/source/l-moninf.cc

```diff
@@ -207,6 +212,94 @@ MIRES1(res_shock, MR_RES_ELEC)
  * @function res_corr
  */
 MIRES1(res_corr, MR_RES_ACID)
+/*** The monster's max HP given in its description.
+ * @treturn string describing the max HP (usually "about X").
+ * @function max_hp
+ */
+static int moninf_get_max_hp(lua_State *ls)
+{
+    MONINF(ls, 1, mi);
+    lua_pushstring(ls, mi->get_max_hp_desc().c_str());
+    return 1;
+}
+/*** The monster's MR level, in "pips" (number of +'s shown on its description).
+ * Returns a value ranging from 0 to 125 (immune).
+ * @treturn int MR level
+ * @function mr
+ */
+static int moninf_get_mr(lua_State *ls)
+{
+    MONINF(ls, 1, mi);
+    lua_pushnumber(ls, ceil(1.0*mi->res_magic()/MR_PIP));
+    return 1;
+}
+/*** Your probability of defeating the monster's MR with a given spell or zap.
```

Comment:
At this place, and others function ends, add a newline before the next function starts.